invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/InvitationResultItemControlled.js

```diff
@@ -82,7 +82,15 @@ export class InvitationResultItemControlled extends Component {
     }
   };
 
-  onView = () => {};
+  onView = () => {
+    const { result } = this.props;
```

Comment:
couldn't this be simply an <a tag instead?

Reply:
you're right :+1:

Reply:
just a comment,please look at my PR I have already solved it